packages/jss/tests/integration/plugins.js

```diff
@@ -387,7 +387,7 @@ describe('Integration: plugins', () => {
       })
       sheet = jss.createStyleSheet({
         a: {
-          color: data => data.color
```

Comment:
how is it possible that this test works? method .update triggers onUpdate only for functions ... hmmm

Reply:
onUpdate is being called for every plugin which implements onUpdate. That's why it works. It's not actually bound to the function plugin which is good.

Reply:
uhh, my bad